src/components/company/CompanyEmissions.astro

```diff
@@ -93,16 +93,16 @@ const scopeEmissionsList = emissions
       <div class="grid gap-4">
         {scopeEmissionsList.map((emission) =>
           emission.value ? (
-            <div class="flex items-center justify-between">
+            <div class={`flex items-center justify-between ${emission.subtitle === 'Totala utsläpp' ? 'text-lg md:text-xl' : ''}`}>
               <div>
-                <h4 class="font-medium">{emission.subtitle}</h4>
+                <h4 class={`font-medium ${emission.subtitle === 'Totala utsläpp' ? 'text-lg md:text-xl' : ''}`}>{emission.subtitle}</h4>
```

Comment:
Use the `cn()` helper and add each conditional class as a comma-separated argument. This will simplify the code.